.gitignore

```diff
@@ -3,4 +3,5 @@ __pycache__
 /.idea
 /conf/config.json
 /data/budabot.db
+/logs/bot.log
```

Comment:
I think it would be better to ignore the entire folder, as long as the .gitkeep continues to work: /logs/